File: kickstart/2022/Coding Practice with Kick Start Session #1/sample_problem.py
def solve(n, m, c):
    sum = 0
    for candy in c:
        sum += candy
    result = sum % m
    return result


if __name__ == '__main__':
    t = int(input())
    for case in range(1, t+1):
        line = list(map(int, input().split()))
        n = int(line[0])
        m = int(line[1])
        c = list(map(int, input().split()))
        result = solve(n, m, c)
        print('Case #{}: {}'.format(case, result))

File: kickstart/2022/Coding Practice with Kick Start Session #1/sample_problem.c
#include <stdio.h>

int main(void) {
    int cases;
    scanf("%d", &cases);
    for (int c=1; c<=cases; c++) {
        int N, M, bag, result;
        scanf("%d", &N);
        scanf("%d", &M);
        int sum = 0;
        for (int i=1; i<=N; i++) {
            scanf("%d", &bag);
            sum += bag;
        }
        result = sum % M;
        printf("Case #%d: %d\n", c, result);
    }
}